apps/web/src/components/user-tools/revision/checkout-reject-buttons.tsx

```diff
@@ -62,7 +62,7 @@ export function CheckoutRejectButtons({
 
       <ModalWithCloseButton
         isOpen={modalMode !== null}
-        onCloseClick={onCloseClick}
+        setIsOpen={(open) => (open ? void null : onCloseClick())}
```

Comment:
Maybe we can have setIsOpen as an optional prop? Then here we could have a cleaner prop,


`open && onCloseClick()`